src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/RawlsDAO.scala

```diff
@@ -17,12 +17,15 @@ trait RawlsDAO extends LazyLogging {
   lazy val rawlsCuratorUrl = FireCloudConfig.Rawls.authUrl + "/user/role/curator"
   lazy val rawlsAdminWorkspaces = FireCloudConfig.Rawls.authUrl + "/admin/workspaces?attributeName=library:published&valueBoolean=true"
   lazy val rawlsWorkspaceACLUrl = FireCloudConfig.Rawls.workspacesUrl + "/%s/%s/acl"
+  lazy val rawlsBucketUsageUrl = FireCloudConfig.Rawls.workspacesUrl + "/%s/%s/bucketUsage"
```

Comment:
I prefer the style of the following next line of code. Use string interpolation instead of a format string with a later call to .format.